funcframework/framework.go

```diff
@@ -142,6 +142,10 @@ func Start(port string) error {
 		}
 	}
 
+	if handler == nil {
+		return fmt.Errorf("no matching function found")
```

Comment:
Could you include the intended function `target` in this error message to help the user understand?